SU2_PY/FSI_tools/FSIInterface.py

```diff
@@ -1247,16 +1239,16 @@ def interpolateSolidPositionOnFluidMesh(self, FSI_config):
           if myid == self.rootProcess:
             for iProc in self.fluidInterfaceProcessors:
               sendBuff = {}
-              for key in self.FluidHaloNodeList[iProc].keys():
-                globalIndex = self.fluidIndexing[key]
+              for key in self.FluidHaloNodeList[iProc].keys():                  # The key is the SU2 global
+                globalIndex = self.fluidIndexing[key]                           # This is the interface global, not the SU2 global
```

Comment:
Not that I would understand what is going on but `# The key is the SU2 global` sounds like there is sth missing.

Reply:
you mean that it should be "the keys are..."?

Reply:
Hm, no I think more sth like 
```python
# The key is the SU2 global >node ID<
```
in case that makes any sense... That is the whole point :) I dont know what `SU2 global` alone should mean. I just had the feeling there was a noun missing (like e.g. `node ID`), not that I know it is missing. 

Treat it more like a question: Is this sentence correct? If yes, then leave it as is 👍

Reply:
I see what you mean. No the comment was actually saying that each key in the dictionary consists of the global ID of an interface node. However, from your question, I feel it was not really clear... I try to clarify

Reply:
Please have a look at #1359